apps/frontend/src/utils/math.ts

```diff
@@ -0,0 +1,50 @@
+import { isBigNumberish } from '@ethersproject/bignumber/lib/bignumber';
+
+import { BigNumberish, BigNumber } from 'ethers';
+import { formatUnits, parseUnits } from 'ethers/lib/utils';
+
+const DEFAULT_UNIT = 18;
+const DEFAULT_DECIMALS = 6;
+
+const unitNames = ['wei', 'kwei', 'mwei', 'gwei', 'szabo', 'finney', 'ether'];
```

Comment:
```suggestion
const units = {
  wei: 0,
  kwei: 3,
  mwei: 6,
  gwei: 9,
  szabo: 12,
  finney: 15,
  ether: 18,
};
```
I'd rather have an object with mapping than just an array.

Reply:
ethers bignumber accepts units as it's defined here so I prefer to use it same way to be fully comply with it.

Reply:
ok then, let's leave it that way